docs/faq.md

```diff
@@ -11,32 +11,30 @@ For value transactions, there is the wallet.rs library which provides a stateful
 
 Additionally, there is also a native C ([iota.c](https://github.com/iotaledger/iota.c)) and an alternative, native javascript ([iota.js](https://github.com/iotaledger/iota.js/tree/chrysalis)) client library. 
 
-## What is Dust protection and how does it work?
+## What is Dust Protection and How Does it Work?
 
-Since IOTA is feeless and has the ability to send microtransactions, attackers could use this to spam the network with very low value transactions, which we call dust. To avoid this we only allow microtransaction below 1Mi of IOTA tokens to another address if you already have at least 1Mi on that address.
+Since IOTA is feeless and has the ability to send microtransactions, attackers could use this to spam the network with very low value transactions, which we call dust. To avoid this, we only allow microtransaction below 1Mi of IOTA tokens to another address if you already have at least 1Mi on that address.
 
 You can read more about Dust Protection [in the RFC here](https://github.com/iotaledger/protocol-rfcs/pull/32).
 
-## What's happening with Coordicide?
+## What is Happening with Coordicide?
 
 This release called Chrysalis still depends on the Coordinator/Compass to run and issue milestones. The next big release after Chrysalis will be Coordicide where we get rid of that dependency. This release is currently in a research/testing phase and will be the main priority after the Chrysalis release.
 
-## When is a transaction on the network considered final/irreversible?
+## When is a Transaction on the Network Considered Final/Irreversible?
 
 Confirmation times on the new network are on average around 10 seconds. Once a transaction is set to confirm it’s final transaction, you don’t have block confirmations like with blockchain.
 
-## Hornet or Bee? Which node software should I use?
+## Hornet or Bee? Which Node Software Should I Use?
 
-You can pick either Bee (Rust based) or Hornet (Go based). We currently recommend Hornet since it’s the more complete version of the node software, Bee is still missing some optional features you might wish to use.
+You can either pick Bee (Rust based) or Hornet (Go based). We currently recommend Hornet since it’s the more complete version of the node software, Bee is still missing some optional features you might wish to use.
```

Comment:
```suggestion
You can either pick [Bee](https://bee.docs.iota.org/) (Rust based) or [Hornet](https://hornet.docs.iota.org/) (Go based). We currently recommend Hornet, since it’s the most complete version of the node software.  Bee is still missing some optional features you might wish to use.
```

- We could also list a couple of the missing optional features here so the user can choose if he needs them or not.

Reply:
I _think_ the original authors intentionally left the optional features blank. Looking at the language around them "although this is probably not useful for IOTA's use case", I'd side with the authors on this one.